### PR TITLE
Make separate Get Set instructions for Back End

### LINKED ISSUE #7401: Get Set for our Back End Development Projects - create simpler version for API Projects
I think Clementine is a wonderful starting place for the Full Stack Apps but many users are finding it to be unnecessary and unhelpful for the API projects.  They are getting tripped up on things like authentication and MVC patterns, which aren't necessary for the API Projects.

Therefore, I'd like to suggest that the Clementine generator instructions are moved to a new step preceding the Full Stack projects.  For the API Projects, either create a different boilerplate or provide none at all.

### PR DESCRIPTION
**Pull request by jondcoleman**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #7401 
#### Description

Move original Clementine based instructions to dynamic web apps section and create
a new waypoint for the API projects.

### CODE REVIEW

**Comment by QuincyLarson** [comment]
I've commented on the issue. Let's discuss this a bit more before accepting the pull request.

**Comment by raisedadead** [comment]
@QuincyLarson @atjonathan I think with the latest comment on the issue thread we can do a QA of this.

**Comment by atjonathan** [comment]
@raisedadead, let's do it :+1:

**Comment by atjonathan** [comment]
@raisedadead, please `QA` this. I will take a look later also.

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 1]
```diff
@@ -5,6 +5,188 @@
   "helpRoom": "HelpBackEnd",
   "challenges": [
     {
+      "id": "bd7158d8c443eddfaeb5bcef",
+      "title": "Get Set for our Back End Development Projects",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]:
```diff
- "title": "Get Set for our Back End Development Projects",
+ "title": "Get Set for our Dynamic Web Application Development Projects",
```

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 2]
```diff
@@ -5,6 +5,188 @@
...
+        [
+          "//i.imgur.com/bjO5pnq.gif",
+          "A gif showing you how to create a new GitHub repository and push your code up to it.",
+          "Create a new GitHub repository. Then copy its .git URL.<br><br>Return to c9.io's terminal and set your GitHub remote URL: <code>git remote set-url origin</code> followed by the URL you copied from GitHub.<br><br>Run <code>git push origin master</code>.<br><br>Now tab back to GitHub and refresh, and you'll see that your code is now on GitHub.",
+          "https://github.com/new"
+        ],
+        [
+          "//i.imgur.com/Qn0K65B.gif",
+          "A gif showing you how to add add-ons to Heroku.",
+          "We will soon add instructions for getting Clementine running on Heroku. For now, develop your Basejumps right on c9.io.",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]: cc/ @FreeCodeCamp/issue-moderators do we want to add this :interrobang:

Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]: @jondcoleman Please remove this section.

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json (hunk 1 above)
Comment (jondcoleman) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]: @atjonathan ok, so is this the only change here?

Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]: @jondcoleman yup :smile:, sorry about the confusion with the other.

**Comment by jondcoleman** [comment]
@atjonathan updates made.

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 3]
```diff
@@ -5,6 +5,188 @@
   "helpRoom": "HelpBackEnd",
   "challenges": [
     {
+      "id": "bd7158d8c443eddfaeb5bcef",
+      "title": "Get Set for our Dynamic Web Application Development",
```
Comment (atjonathan) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]: ~~Dynamic Web Application Development~~ Dynamic Web Application **Projects**

Comment (jondcoleman) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]: Haha, ok, coming up

**Comment by atjonathan** [comment]
@jondcoleman, could you squash your commits? (https://www.freecodecamp.com/wiki/en/git-squash/). Thanks :smile:

**Comment by jondcoleman** [comment]
@atjonathan squashed :)

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing :tada:

**Pull request merged**